### DISCUSSION TITLE
Can you build an affiliate system on top of Hyperswitch like Rewardful?

### DISCUSSION THREAD

**Discussion by deadcoder0904** [juspay/hyperswitch#2397] [Q&A]
I am currently integrating Stripe & Paypal into my Next.js project.

Was wondering if you guys have implemented a simple way to do affiliate or coupon system on top of Hyperswitch? Or has anyone using Hyperswitch done that.

This is a must for me to give it a shot.

**Comment by deadcoder0904** [comment]
@manojradhakrishnan why did you close this one? is it already built-in or not gonna be the focus for a long time?

**Reply by NarsGNA** [reply]
Hey @deadcoder0904 sorry we could not fit this onto our current roadmap as we are focusing on open sourcing our control center, web client and other services

**Reply by deadcoder0904** [reply]
@NarsGNA ahh cool, but would love to know if you guys have APIs that enable it? I mean I wanna use Hyperswitch for my products but referral system is a must-have for me. Already integrated 2 payment providers & gonna integrate more so would be curious if there's a timeline to this. Or not gonna focus on it for 1 year or so?
